zulip_bots/zulip_bots/bots/dropbox_share/test_dropbox_share.py

```diff
@@ -36,6 +51,9 @@ def search_files(*args, **kwargs):
         )
     ])
 
+def empty_search(*args, **kwargs):
```

Comment:
Minor nitpicking, it should probably be `get_empty_search_result` looking at how other functions are named.

Reply:
@aero31aero Renamed function to `get_empty_search_result`